torch/lib/TH/generic/THTensorRandom.c

```diff
@@ -23,6 +23,16 @@ void THTensor_(random)(THTensor *self, THGenerator *_generator)
 #endif
 }
 
+void THTensor_(clampedRandom)(THTensor *self, THGenerator *_generator, long min, long max) {
+  THArgCheck(max > min, 2, "upper bound must be greater than lower bound");
+  TH_TENSOR_APPLY(real, self, *self_data = (real)((THRandom_random(_generator) % (max - min)) + min);)
+}
+
+void THTensor_(cappedRandom)(THTensor *self, THGenerator *_generator, long max) {
+  THArgCheck(max > 0, 1, "upper bound must be positive");
```

Comment:
We should change the error message to use the same names as the variable i.e. `upper bound` -> `max` or vise versa